bigquery_etl/cli/utils.py

```diff
@@ -78,29 +79,31 @@ def table_matches_patterns(pattern, invert, table):
     return matching != invert
 
 
-def paths_matching_checks_pattern(pattern, sql_path, project_id):
+def paths_matching_checks_pattern(
+    pattern: str, sql_path: Optional[str], project_id: Optional[str]
+) -> Iterator[Tuple[Path, str, str, str]]:
     """Return single path to checks.sql matching the name pattern."""
     checks_files = paths_matching_name_pattern(
         pattern, sql_path, project_id, ["checks.sql"], CHECKS_FILE_RE
     )
 
-    if len(checks_files) == 1:
-        match = CHECKS_FILE_RE.match(str(checks_files[0]))
+    for checks_file in checks_files:
+        match = CHECKS_FILE_RE.match(str(checks_file))
         if match:
             project = match.group(1)
             dataset = match.group(2)
             table = match.group(3)
-        return checks_files[0], project, dataset, table
+        yield checks_file, project, dataset, table
     else:
         print(f"No checks.sql file found in {sql_path}/{project_id}/{pattern}")
         return None, None, None, None
 
 
 def paths_matching_name_pattern(
     pattern, sql_path, project_id, files=["*.sql"], file_regex=QUERY_FILE_RE
-):
+) -> List[Path]:
```

Comment:
I wanted typing in the above function, and `mypy` complained unless I added it here as well.